upgrade-notes/storefront_20251030_055644.md

```
@@ -0,0 +1,12 @@
+#### Updated Reset password page ([#4254](https://github.com/shopsys/shopsys/pull/4254))
+
+- updated the info page shown after submitting a password reset request
+- created a new `PageHero` component and added it to several pages across the storefront, and updated skeletons accordingly
+- refactored form error handling
+    - removed the error popup from most forms
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)